docs/service_upgrade.md

```diff
@@ -0,0 +1,12 @@
+## Changning CoreDNS service type
+Since Kubernetes API doesn't allow to change service type, following steps should be taken.
+Option 1:
+* deploy new serivce with changed name by setting `coredns.service.name=migrate-svc`
+  ```helm upgrade -n <k8gb namespace> --install <release name> <chart repo>/k8gb --set coredns.service.name=migrate-svc```
+* deploy back with original service name
+  ```helm upgrade -n <k8gb namespace> --install <release name> <chart repo>/k8gb```
```

Comment:
![image](https://user-images.githubusercontent.com/518532/115539689-8df2d900-a29d-11eb-8ea4-78f4feaf8e88.png)
It's not easily readable after markdown rendering, please break the lines up

Reply:
done